assets/editorial_features.styl

```diff
@@ -1,3 +1,3 @@
 @require '../components/main_layout/stylesheets'
 @require '../components/article/stylesheets'
-@require '../apps/editorial_features/components/eoy'
+@require '../apps/editorial_features/components/eoy/stylesheets/eoy'
```

Comment:
Cleared this up -- I forgot that it will look for `index.styl`, when you give it the parent folder, so we renamed `eoy.styl` to `index.styl`